mode/examples/Topics/File IO/LoadFile1/LoadFile1.pyde

```diff
@@ -0,0 +1,31 @@
+"""
+LoadFile 1
+
+Loads a text file that contains two numbers separated by a tab ('\t').
+A new pair of numbers is loaded each frame and used to draw a point on the screen.
+"""
+
+index = 0
+
+
+def setup():
+    size(200, 200)
+    background(0)
+    stroke(255)
+    frameRate(12)
+    global lines
+    lines = loadStrings("positions.txt")
+
+
+def draw():
+    global index
+    pieces = []
+    if (index < len(lines)):
+        pieces = split(lines[index], '\t')
+        if (len(pieces) == 2):
+            x = int(pieces[0]) * 2
```

Comment:
I took this as is from the example in the Java mode. I think this was done because the inputs are in the range of 0-99 whereas the canvas was of size 200 X 200. Should I remove it?